test/integration/targets/mariadb_replication/tasks/mariadb_replication_initial.yml

```diff
@@ -0,0 +1,72 @@
+# Copyright: (c) 2019, Andrew Klychkov (@Andersson007) <aaklychkov@mail.ru>
```

Comment:
Is `mariadb_replication` tested already by this target?

Reply:
no, i made it from scratch (there aren't any ci for mysql_replication now)

Reply:
i added checks for 3 basic options